_posts/2022-12-21-easily-verify-your-rust-in-ci-with-kani.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Easily verify your Rust in CI with Kani and Github Actions"
+---
+
+Formal verification tools, such as Kani, are valuable because they provide a way to definitively check, using mathematical techniques, whether a property of your code is true under all circumstances.
+To put it colloquially: testing is great for catching the **first** bug, formal verification is great for knowing you’ve caught the **last** bug.
+To learn more about Kani, and how you can use it to prevent bugs in your Rust code, you can [read the Kani documentation](https://model-checking.github.io/kani/), or [consult the real-world examples on our blog](../../..).
+
+Code, however, is seldom static.
+As code evolves, how can we ensure that it remains correct; i.e.
+how do we catch the **next** bug?
+In the testing world, this is where **Continuous Integration** (CI) shines: by ensuring that tests are re-run on all code-changes, CI enables developers to catch bugs before they impact users.
+Many platforms, such as GitHub, provide facilities to automatically incorporate CI into existing development workflows.
+For example, [GitHub Actions](https://github.com/features/actions) provides developers with a facility to run checks, ranging from tests, to linters, to code-formatters, and have the results displayed on the pull request (PR) in real time.
+Repositories can even [require that certain checks pass before merging a PR](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-status-checks-before-merging).
+For example, on our own Kani repo, a CI status report on a PR might look like this:
+
+<img src="{{site.baseurl | prepend: site.url}}/assets/images/kani-ci-checks.png" alt="Kani CI Checks" />
+
+Similarly, **Continuous Verification** enables developers to prevent regressions before they happen.
```

Comment:
See comment above. Apart from that, I'm not sure that emphasizing "continuous verification" new term is warranted. It's not used anywhere else in the blog post, and I don't think we're particularly advocating this term (or are we?), so it seems distracting to introduce a new term